src/app/shared/components/xterm-terminal/terminal-toolbar/style.scss

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@use 'variables';
+@use 'mixins';
+
+.toolbar-container {
+  @include mixins.size(100%, 4rem);
+
+  align-items: center;
```

Comment:
It would be nice to use flex-layout in the template for this.

Reply:
removed